tools/test.mk

```diff
@@ -972,25 +972,25 @@ test-size-all:
 	make keysclean
 	make test-size SIGN=ED25519 LIMIT=11380
 	make keysclean
-	make test-size SIGN=ECC256  LIMIT=17696
+	make test-size SIGN=ECC256  LIMIT=17824
```

Comment:
Is it possible to add a GitHub action that also does an update to latest master in the submodules to detect these issues earlier?

Reply:
Let's discuss about this, this should perhaps be in the wolfSSL CI instead, so we can spot it while reviewing PRs?